proxy/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: 1.0.0
```

Let's keep these in sync with the proxy version.  
```suggestion
appVersion: 1.4.0
```